Python/emscripten_trampoline.c

```diff
@@ -43,7 +43,7 @@ EM_JS(int, _PyEM_CountFuncParams, (PyCFunctionWithKeywords func),
     if (n !== undefined) {
         return n;
     }
-    n = WebAssembly.Function.type(wasmTable.get(func)).parameters.length;
+    n = wasmTable.get(func).type().parameters.length;
```

Comment:
Can we use preprocessor to detect Emscripten version in here?

Reply:
It doesn't depend on the Emscripten version, it depends on the JavaScript runtime version. We could potentially feature detect this by checking whether `WebAssembly.Function.type` is defined. This is probably a good idea actually so that we can support node v20 as well as node v22.

Reply:
I prefer the way we can support multiple version as possible. You can predetection the function through the configure.ac

Reply:
I switched to a version that works on both the old node and the new one. Manually tested. Thanks for the suggestion @corona10!